src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entry.html.twig

```diff
@@ -23,13 +29,14 @@
             <li><a href="mailto:hello@wallabag.org?subject=Wrong%20display%20in%20wallabag&amp;body={{ entry.url|url_encode }}" title="{% trans %}Does this article appear wrong?{% endtrans %}" class="tool bad-display icon icon-delete"><span>{% trans %}Does this article appear wrong?{% endtrans %}</span></a></li>
         </ul>
     </div>
-    <div id="article">
-        <header class="mbm">
-            <h1>{{ entry.title|raw }} <a href="{{ path('edit', { 'id': entry.id }) }}" title="{% trans %}Edit tags{% endtrans %}">✎</a></h1>
-        </header>
+
         <aside class="tags">
-            {% for tag in entry.tags %}<span class="mdi-action-label-outline">{{ tag.label }}</span>{% endfor %}
-            {{ render(controller( "WallabagCoreBundle:Tag:addTagForm", { 'id': entry.id } )) }}
+            {% for tag in entry.tags %}
+                <span class="mdi-action-label-outline">{{ tag.label }}</span><a href="{{ path('remove_tag', { 'entry': entry.id, 'tag': tag.id }) }}"><i>✘</i></a>
+                {% endfor %}
```

Comment:
Tabulation problem, also could you add a space after the span? It'll avoid the _cross_ to be attached to the tag

![capture d ecran 2016-02-15 a 14 29 38](https://cloud.githubusercontent.com/assets/62333/13049939/945b49ac-d3f0-11e5-9e50-e82051b15e68.png)